packages/python-aiohttp/python-aiohttp.spec

```diff
@@ -3,7 +3,7 @@
 
 Name:           python-%{pypi_name}
 Version:        3.7.4
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
This won't work, as 3.7.4-2 already exists (and is the py38 rebuild)

We could version this 3.7.4-1.1?

Reply:
The alternative is to leave this fix out of 3.14. In Foreman it doesn't really matter since Apache depends on the same file and we don't support running without Apache.

Reply:
works for me!